pan3d/ui/preview_bounds.py

```diff
@@ -0,0 +1,14 @@
+from trame_client.widgets.core import HtmlElement
```

Comment:
This file should be in `pan3d/ui/widgets.py`

Reply:
Do you mean just renaming `preview_bounds` -> `widgets`? Or should it be moved as well?

Reply:
just rename it.